app/styles/main.scss

```diff
@@ -1,97 +1,8109 @@
 // NHS.UK frontend
-@import '../../node_modules/nhsuk-frontend/packages/nhsuk';
+@import '../../node_modules/nhsuk-frontend/packages/nhsuk.scss';
+
+
 
 /* ==========================================================================
    #SIDE-NAV
    ========================================================================== */
 
 .app-side-nav__heading {
-  @include nhsuk-font(19, $line-height: 1.25, $weight: bold);
```

Comment:
We should keep these mixins from the NHS.UK frontend library opposed to rewriting the styles out. We will benefit from any improvements to the NHS.UK frontend library without having to update our code here everytime.